cmd/internal/container/mesos/handler_test.go

```diff
@@ -22,7 +22,6 @@ import (
 	containerlibcontainer "github.com/google/cadvisor/container/libcontainer"
 	"github.com/google/cadvisor/fs"
 	info "github.com/google/cadvisor/info/v1"
-	"github.com/mesos/mesos-go/api/v1/lib"
```

Comment:
I found CI error after removed this line:

```
Error: vet: internal/container/mesos/handler_test.go:31:21: undeclared name: mesos
Error: make: *** [Makefile:71: vet] Error 2
Error: Process completed with exit code 2.
```

But I have no idea about the reason. It seems that the mesos lib didn't exist anymore, see [mesos-go](https://github.com/mesos/mesos-go/tree/master/api).